aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -218,7 +219,80 @@ void qmaxpool_2d_nhwc_kernel(const Tensor &qx,
       }  // for row
     }  // for b
   });
-} 
+}
+
+void qtopk_kernel(Tensor& values,
```

Comment:
can't you literally use the existing function and define _isnan<quint8> to be always false? but I'm fine with duplication as is as well

Reply:
The current macros we have for dispatch are `AT_DISPATCH_ALL_TYPES` and `AT_DISPATCH_QINT_TYPES`. The former does not include qint types and I didn't want to add yet another dispatch macro so I just copied the code here